.github/ISSUE_TEMPLATE/stale.yml

```diff
@@ -0,0 +1,27 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 30
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - Feature
+  - Enhancement
+  - Bug
+  - NHN
+
+# Label to use when marking as stale
+staleLabel: inactive
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as inactive because there hasn’t been much going on it lately.
+  It is going to be closed after 7 days. Thanks!
+
+# Comment to post when closing a stale Issue or Pull Request.
+closeComment: >
+  This issue will be closed due to inactivity. Thanks for your contribution!
```

Comment:
This file has changed path [here](https://github.com/nhn/tui.editor/tree/master/.github). I think you fork before the file path changes. Please remove this file.